OCP 4.19 Release Notes for P/Z (#93142)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

This release introduces support for the following features on {ibm-power-title}:

* Expand Compliance Operator Profile Support - PCI-DSS v4
* Node Feature Discovery support for nodes with IBM Power11
* Support for IBM Power11

[id="ocp-release-notes-ibm-z_{context}"]
=== {ibm-z-title} and {ibm-linuxone-title}

The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.

This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:

> * Support for {ibm-z-name} z17 and {ibm-linuxone-name} 5

Reviewer: This should resolve in "IBM z17" not "IBM Z z17"
```suggestion
* Support for {ibm-name} z17 and {ibm-linuxone-name} 5
```

> * Support for IBM z17 and IBM LinuxONE 5
> * Hardware-based IBM Crypto Express (CEX) encryption

Reviewer: ```suggestion
* Support for {ibm-z-name} z17 and {ibm-linuxone-name} 5
* Linux Unified Key Setup (LUKS) encryption via {ibm-name} Crypto Express (CEX)
```

This release introduces support for the following features on {ibm-power-title}:

> * Expand Compliance Operator Profile Support - DISA STIG

Reviewer 2: DISA STIG is a profile that aligns with the Defense Department's security.

> * Expand Compliance Operator Profile Support - DISA STIG

Reviewer: I would write out at first mention @mburke5678 would this fit? 
```suggestion
* Expand Compliance Operator support with profiles for Defense Information Systems Agency Security Technical Implementation Guide (DISA STIG)
```

[id="ocp-release-notes-ibm-z_{context}"]
=== {ibm-z-title} and {ibm-linuxone-title}

> The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.

Reviewer: ```suggestion
The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
```

This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:

* Support for {ibm-name} z17 and {ibm-linuxone-name} 5

> * Linux Unified Key Setup (LUKS) encryption via {ibm-name} Crypto Express (CEX)

Reviewer: ```suggestion
* Boot volume Linux Unified Key Setup (LUKS) encryption via {ibm-name} Crypto Express (CEX)
```
